## Andy Benson Seminar Notes

Gave presentation at Michigan State University Microbiology & Molecular Genetics Department Seminar on December 3rd, 2012.

Talk was entitled "Assembly of the gut microbiome is shaped by convergence of genetic and environmental factors"

Andy is a [professor in the Food Science & Technology Department](http://foodsci.unl.edu/foodsci/abenson) at the University of Nebraska

See review article by Zhao 2013 in Nature Review Microbiolgy [The gut microbiota and obesity: from correlation to causality](http://www.nature.com/nrmicro/journal/v11/n9/abs/nrmicro3089.html)

Other Papers of Interest include:

Human Microbiome Project Consortium 2012 Nature Paper [Structure, function and diversity of the healthy human microbiome](http://www.nature.com/nature/journal/v486/n7402/full/nature11234.html)

Arumugam <i>et al</i> 2011 [Enterotypes of the human gut microbiome](http://www.nature.com/nature/journal/v473/n7346/abs/nature09944.html)

Jia <i>et al</i> 2008 [Gut microbiota: a potential new territory for drug targeting](http://www.nature.com/nrd/journal/v7/n2/abs/nrd2505.html)

Cho & Blaser [The human microbiome: at the interface of health and disease](http://www.nature.com/nrg/journal/v13/n4/abs/nrg3182.html)

Qin <i>et al</i> 2010 [A human gut microbial gene catalogue established by metagenomic sequencing](http://www.nature.com/nature/journal/v464/n7285/full/nature08821.html)

Kuczynski <i>et al</i> 2012 [Experimental and analytical tools for studying the human microbiome](http://www.nature.com/nrg/journal/v13/n1/abs/nrg3129.html)

Review from Eloe-Fadrosh & Rasko 2013 [The Human Microbiome: From Symbiosis to Pathogenesis](http://www.annualreviews.org/doi/abs/10.1146/annurev-med-010312-133513)

Andy specifically referenced tools developed in [Curtis Huttenhower's Lab](http://huttenhower.sph.harvard.edu/).

One of the goals of the research is to establish a human gut microbiome gene catalogue through Metagenomic sequencing

Gene by diet interactions were determined in mouse models and they are interested in expanding this study to cover the large amount of data they have on cattle gut microbiomes.

Some assemblers that were used on their data include: [IDBA-UD](http://i.cs.hku.hk/~alse/hkubrg/projects/idba_ud/), [metaVelvet](http://metavelvet.dna.bio.keio.ac.jp/)

Establishing a [limit of detection](http://en.wikipedia.org/wiki/Detection_limit) using metagenomics techniques


